codejail/tests/test_safe_exec.py

```diff
@@ -48,12 +48,12 @@ def test_complex_globals(self):
             textwrap.dedent("""\
             from builtins import bytes
             test_dict = {1: bytes('a', 'utf8'), 2: 'b', 3: {1: bytes('b', 'utf8'), 2: (1, bytes('a', 'utf8'))}}
-            foo = "bar"
-            test_dict_type = type(test_dict)
+            bad_val = b'\\x99'
             """),
             globs
         )
         self.assertDictEqual(globs['test_dict'], {'1': 'a', '2': 'b', '3': {'1': 'b', '2': [1, 'a']}})
+        assert 'bad_val' not in globs
 
```

Comment:
We should add some direct tests of json_safe.  I'd like to see some sample data tested to see that json_safe works the way we need it to.